troubadix/plugins/valid_oid.py

```diff
@@ -81,12 +82,11 @@ def check_content(
             )
             return
 
+        family_pattern = get_special_script_tag_pattern(SpecialScriptTag.FAMILY)
+        family_match = family_pattern.search(file_content)
+
         # Vendor-specific OIDs
         if "1.3.6.1.4.1.25623.1.1." in oid:
-            family_pattern = get_special_script_tag_pattern(
-                SpecialScriptTag.FAMILY
-            )
-            family_match = family_pattern.search(file_content)
             if family_match is None or family_match.group("value") is None:
```

Comment:
Move this outside of the "if" check as well and then drop the duplicated check in the newly added `# Fixed OID-scheme for Windows OIDs` as well?

Reply:
So something like this (outside of the `if()`) won't work?

```
        family_pattern = get_special_script_tag_pattern(SpecialScriptTag.FAMILY)
        family_match = family_pattern.search(file_content)
        if family_match is None or family_match.group("value") is None:
            yield LinterError(
                "VT is missing a script family!",
                file=nasl_file,
                plugin=self.name,
            )
            return

        # Vendor-specific OIDs
        if "1.3.6.1.4.1.25623.1.1." in oid:
```